db/version_set.cc

```diff
@@ -2085,6 +2182,10 @@ void Version::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
   uint64_t num_data_read = 0;
   uint64_t num_sst_read = 0;
 
+  MultiGetRange keys_with_blobs_range(*range, range->begin(), range->end());
```

Comment:
Do we use this `MultiGetRange` somewhere?

Reply:
It's used in `Version::MultiGetBlob()` for `range.AddValueSize()`.

Reply:
Right, thanks :)